### ISSUE TITLE
[Typescript] Could not find a declaration file for module unform-material-ui

### ISSUE THREAD

**Issue by jairokoning** [italoiz/unform-community-packages#29]
I try to use the unform-material-ui with typescript but could not find a declaration file for module unform-material-ui. I even try to use @types/index.d.ts and add  "declare module 'material-unform-ui';", but too not working.

![unform-material-ui_types-error](https://user-images.githubusercontent.com/29260158/90689844-11f44d00-e247-11ea-8211-8b82321bd51a.jpg)

**Comment by italoiz** [comment]
Hi, @jairokoning Can you provide a repository with this issue to reproduce? I use this library in my projects and do not have this problem with typings.

**Comment by guitexa** [comment]
Hi @italoiz, I have the same problem, I'm using `unform-material-ui version 1.0.4`

Because this project is private, I can't share to reproduce, if you know what's happening let me know please.

Thanks!!

**Comment by italoiz** [comment]
@guitexa What version of Typescript are using?

**Comment by guitexa** [comment]
> 
> 
> @guitexa What version of Typescript are using?

I was using `typescript: ~3.7.2`, I've upgraded to `typescript: ~4.0.2` but it doesn't works

The error is
```
Could not find a declaration file for module 'unform-material-ui'. 'C:/Development/frontend/node_modules/unform-material-ui/dist/index.js' implicitly has an 'any' type.
  Try `npm install @types/unform-material-ui` if it exists or add a new declaration (.d.ts) file containing `declare module 'unform-material-ui';`  TS7016
```

I'm using this element
```js
import { TextField } from 'unform-material-ui';
```

Is right I delete the `import { TextField } from '@material-ui/core'` isn't? I developed with `@material-ui/core` and now I'd like to replace for `unform-material-ui` and I've changed only the imports, I don't know if I'm right

**Comment by guitexa** [comment]
> 
> 
> @guitexa What version of Typescript are using?

If you wish to verify, I've created a SandBox with Typescript 4.0.3.
https://codesandbox.io/s/vibrant-shannon-hm3bl

**Comment by italoiz** [comment]
@guitexa A release was published `v1.0.5`. Try to update the library and test your code again, please. Thank you for your feedback.

**Comment by guitexa** [comment]
Updated and work fine, thank you so much for helping with that 🙏🏼🙏🏼
